ansible/README.md

```diff
@@ -0,0 +1,178 @@
+use a plaintext (not encrypted) file to redirect to the vault file for
+easier maintenance, as described in [the
+documentation](https://docs.ansible.com/ansible/2.5/user_guide/playbooks_best_practices.html#best-practices-for-variables-and-vaults).
+For this playbook, copy the example vault file and fill it in with your
+own secrets, then vault encrypt it.
+
+```
+    $ cp ansible/mediawiki_vars/vault.yml.tmpl ansible/mediawiki_vars/vault.yml
+    # Edit vault file and fill in your passwords
+    # Encrypt the file
+    $ ansible-vault encrypt mediawiki_vars/vault.yml
+    New Vault password:
+    Confirm New Vault password:
+
+    Encryption successful
+```
+
+Then, when you run the playbook, you'll use the `--ask-vault-pass` flag.
+
+    $ ansible-playbook mediawiki-install.yml --ask-vault-pass
+    Vault password:
+
+If you want to run the command in a script, you can also pass the vault
+password in a file.
+
+## Server configuration
+
+We include an `mwiki-default.conf.tmpl` in this directory.  You will need to copy that to your .conf file like so:
+
+    $ cp mwiki-default.conf.tmpl mwiki-default.conf
+    # Edit ServerName to be whatever you want to call your host
+
+If you know what changes you want to make to your Apache configuration
+(e.g. adding SSL), you can make those here.  You will at least need
+to fill in the Server Name.
+
+## Mediawiki Farm (i.e., multiple wikis)
+
+Our configuration for multiple wikis is currently closely based on
+[documentation in another
+branch](https://github.com/OpenTechStrategies/librehq-wikis/blob/mediawiki-updates/MEDIAWIKI_INSTALL.md)
+of this repository.  We expect these to be united on the master branch
+in the future.
+
+To use the mediawiki farm parts of this Ansible playbook, you will need
+to add the database password to `test1.yml` and `test2.yml`.
+
+    $ cp mediawikifarmdocs/test1.yml.tmpl mediawikifarmdocs/test1.yml
+    # Edit database password to match the one you entered in your vault
+
+
+## Local testing
+
+Locally, Ansible works well with Vagrant.
+
+    $ sudo apt-get install vagrant
+    $ sudo apt-get install virtualbox
+
+Note that the [Vagrant
+documentation](https://www.vagrantup.com/docs/installation/) suggests
+_not_ using the version of Vagrant packaged by your operating system,
+but rather using the more complete and up-to-date versions they
+[provider](https://www.vagrantup.com/downloads.html). Either way,
+downloading may take a few minutes.  If you choose to download the
+`.deb` file, you can install it with `sudo dpkg -i
+/path/to/vagrant_2.2.1_x86_64.deb`. Test your local installation with
+the following commands:
+
+    # Adds a Debian image to your VirtualBox installation
+    $ vagrant box add debian/testing64
+
+    # Since there is a committed Vagrantfile in this repo, you don't
+    # need to `vagrant init` and can instead move on to these commands:
+    
+    $ emacs Vagrantfile
+      # Add: config.vm.box = "debian/testing64"
```

Comment:
This was already in the Vagrantfile, so I didn't have to add it.  Maybe change this to just say 'confirm this line is there' or similar.